packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts

```diff
@@ -142,6 +143,7 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
     }
 
     private selectImage(editor: IEditor, image: HTMLImageElement) {
+        this.suspendSelectionChangeEvent = true;
         editor.setDOMSelection({
             type: 'image',
             image: image,
```

Comment:
Can we set suspendSelectionChangeEvent = false after here?

Reply:
if we set false after here, it the selection will be removed when selectionchanged happened